packages/paste-core/components/progress-steps/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,171 @@
+import * as React from 'react';
+import {render} from '@testing-library/react';
+import {CustomizationProvider} from '@twilio-paste/customization';
+
+import {
+  ProgressSteps,
+  ProgressStepSeparator,
+  ProgressStepComplete,
+  ProgressStepCurrent,
+  ProgressStepError,
+  ProgressStepIncomplete,
+} from '../src';
+import {Divs, Buttons, Anchors} from '../stories/horizontal.stories';
+
+describe('ProgressSteps', () => {
+  it('should render divs correctly', () => {
+    const {queryAllByText} = render(<Divs />);
+    const [completeIcon, completeLabel] = queryAllByText('Complete');
+    const [incompleteIcon, incompleteLabel] = queryAllByText('Incomplete');
+    const [errorIcon, errorLabel] = queryAllByText('Error');
+    const [currentIcon, currentLabel] = queryAllByText('Current');
+
+    expect(completeIcon).toBeDefined();
+    expect(completeLabel).toBeDefined();
+    expect(incompleteIcon).toBeDefined();
+    expect(incompleteLabel).toBeDefined();
+    expect(errorIcon).toBeDefined();
+    expect(errorLabel).toBeDefined();
+    expect(currentIcon).toBeDefined();
+    expect(currentLabel).toBeDefined();
+
+    expect(completeLabel.parentElement?.tagName).toEqual('DIV');
+  });
+
+  it('should render buttons correctly', () => {
+    const {getByText} = render(<Buttons />);
+    const signupLabel = getByText('Sign up').parentElement;
```

Comment:
suggest: I learnt you can do this the other day `getByRole('button', {name: 'Sign up'})` rather than travel up the tree

Reply:
True, way better. Will update this